jetty-home/src/main/assembly/jetty-assembly.xml

```diff
@@ -15,6 +15,8 @@
         <exclude>**/META-INF/**</exclude>
         <exclude>*-config.jar</exclude>
       </excludes>
+      <fileMode>0444</fileMode>
+      <directoryMode>0755</directoryMode>
```

Comment:
I think the following would be better:
```suggestion
      <fileMode>0400</fileMode>
      <directoryMode>0700</directoryMode>
```

We should not be granting any permissions to groups or others

Reply:
Also should we set `0800` for the files in bin?

Reply:
good points.
I was rather happy that these modes I started with had no issue in the `/test-distribution/` already.
Wonder if we should expand those tests to use the `bin/jetty.sh` too (in some regard)?